### PR TITLE
misc typos in json.

### PR DESCRIPTION
**Pull request by cainiaowu**
#### Summary

```SUMMARY: I18N "misc typos in json files."```  
#### Purpose of change

Unify some punctuations in json.

### CODE REVIEW

File: data/json/mutations.json [hunk 1]
```diff
@@ -4485,7 +4485,7 @@
         "points" : 2,
         "visibility" : 4,
         "ugliness" : 5,
-        "//~" : "The idea is that you, for lack of a better term, plant yourself.  Crafting and digging don't count because they presume a certain amount of movement during the task.  The handheld game does count.",
+        "//" : "The idea is that you, for lack of a better term, plant yourself.  Crafting and digging don't count because they presume a certain amount of movement during the task.  The handheld game does count.",
```
Comment (Qrox) [inline data/json/mutations.json]: Same here

File: data/json/items/tool_armor.json [hunk 2]
```diff
@@ -2299,7 +2299,7 @@
             ]
         },
         "symbol" : "&",
-        "//~" : "ol' is an informal short-form for old, and here is part of big-ol as large-and-cumbersome.",
+        "//" : "ol' is an informal short-form for old, and here is part of big-ol as large-and-cumbersome.",
```
[the remark this answers is not in the document]
Comment (cainiaowu) [reply data/json/items/tool_armor.json]: In the cpp files only, I dont think the Python script can extract it in this form.

[the remark this answers is not in the document]
Comment (ZhilkinSerg) [reply data/json/items/tool_armor.json]: Regardless `//~` is not a typo, but a comment for translators.

File: data/json/items/chemicals_and_resources.json [hunk 3]
```diff
@@ -76,7 +76,7 @@
     "name_plural" : "limestone",
     "symbol" : "=",
     "color" : "dark_gray",
-    "description" : "A handful of small chunks of limestone.  Useful for experiments at science fairs, but pretty useless as ammo.  Maybe something could be crafted from this stuff…",
+    "description" : "A handful of small chunks of limestone.  Useful for experiments at science fairs, but pretty useless as ammo.  Maybe something could be crafted from this stuff...",
```
Comment (cake-pie) [inline data/json/items/chemicals_and_resources.json]: How is proper ellipsis character as opposed to three periods a "typo"?
Ellipsis already used for [shortening strings to fit UI](https://github.com/CleverRaven/Cataclysm-DDA/blob/419dca391d1f2a8c2f953bfa084454ee15901594/src/catacharset.cpp#L616).

File: data/json/items/tool_armor.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (cainiaowu) [reply data/json/items/tool_armor.json]: the translators cant see this comment as it is not extracted by the python script.
image

File: data/json/items/chemicals_and_resources.json (hunk 3 above)
Comment (cainiaowu) [reply data/json/items/chemicals_and_resources.json]: In other description/mission text, Ellipsis is replaced with three commas instead.

Also ellipsis is an ambiguous-width characters, which will cause issue like #12474.

**Pull request merged**